testing/distrobuilder/APKBUILD

```diff
@@ -0,0 +1,73 @@
+# Contributor: Carlo Landmeter <clandmeter@alpinelinux.org>
+# Maintainer:
+pkgname=distrobuilder
+pkgver=0_git20181004
+_commit=d2329be9569d45028a38836186d2353b8ddfe1ca
+pkgrel=0
+pkgdesc="System container image builder"
+url="https://linuxcontainers.org/"
+arch="all"
+license="Apache-2.0"
+depends="xz gnupg1"
+makedepends="go linux-headers glide"
+subpackages="$pkgname-lxc::noarch $pkgname-lxd::noarch"
+options="!check" # has no check
+source="$pkgname-$pkgver.tar.gz::https://github.com/lxc/distrobuilder/archive/$_commit.tar.gz
+	glide-$pkgver.yaml
+	glide-$pkgver.lock
```

Comment:
i think go modules is replacing glide, so we probably want to use go modules instead.

Reply:
We don't have a proper way to handle go modules atm so its better to use glide for now.
We first need to write something to the ml on how to use them properly.